src/main/java/lotto/domain/LottoValidate.java

```diff
@@ -5,13 +5,21 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 public class LottoValidate {
     private static final int REFERENCE_PRICE = 1000;
     private static final int REMAINDER_VALUE = 10;
+    private static final int MAX_LENGTH = 6;
 
     private int count = 0;
 
+    public static void duplicateValidate(Set<Integer> lottos) {
+        if(lottos.size() != MAX_LENGTH) {
```

Comment:
`Set`을 이용한 중복제거 좋은데요 👍 
컨벤션만 다시 한번 확인하고 가시면 될 것 같습니다 😄 
```suggestion
        if (lottos.size() != MAX_LENGTH) {
```